services/datamanager/builtin/file_deletion.go

```diff
@@ -0,0 +1,139 @@
+package builtin
+
+import (
+	"context"
+	"errors"
+	"io/fs"
+	"math"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ricochet2200/go-disk-usage/du"
+
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/services/datamanager/datacapture"
+	"go.viam.com/rdk/services/datamanager/datasync"
+)
+
+var (
+	fsThresholdToTriggerDeletion = .90
+	captureDirToFSUsageRatio     = .5
+	deleteEveryNth               = 5
+)
+
+var errAtSizeThreshold = errors.New("capture dir is at correct size")
```

Comment:
maybe `capture directory has reached or exceeded disk usage threshold for deletion`?

Reply:
done!